yarrrml/.htaccess

```
@@ -11,6 +11,8 @@ RewriteRule ^matey\/screencast-target http://rml.io/yarrrml/matey/screencast-tar
 RewriteRule ^matey\/screencast-target/ http://rml.io/yarrrml/matey/screencast-target.html [R=302,L]
 RewriteRule ^matey\/screencast http://rml.io/yarrrml/matey/screencast.html [R=302,L]
 RewriteRule ^matey\/screencast/ http://rml.io/yarrrml/matey/screencast.html [R=302,L]
+RewriteRule ^matey\/repository http://github.com/RMLio/matey [R=302,L]
+RewriteRule ^matey\/repository/ http://github.com/RMLio/matey [R=302,L]
```

FYI:
- simplify all these rule pairs to one line by making the traliing `/` optional
- no need to escape the middle path `/`
- please use `https` for github since it's more secure and their server will always do that redirect anyway
```suggestion
RewriteRule ^matey/repository/?$ https://github.com/RMLio/matey [R=302,L]
```